src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1087,7 +1087,10 @@ List<Exception> generalResolutionExceptions
                             bool logWarning = idealAssemblyRemappingsIdentities.Any(i => i.assemblyName.FullName.Equals(fusionName) && i.reference.GetConflictVictims().Count == 0);
                             StringBuilder logConflict = StringBuilderCache.Acquire();
                             LogConflict(conflictCandidate, fusionName, logConflict);
-                            StringBuilder logDependencies = logWarning ? logConflict.AppendLine() : StringBuilderCache.Acquire();
+
+                            // If we logging warnings append it into existing StringBuilder, otherwise build details by new StringBuilder.
```

Comment:
```suggestion
                            // If we are logging warnings append it into existing StringBuilder, otherwise build details by new StringBuilder.
```